bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -93,8 +93,11 @@ thing-type.shelly.shellyplusrgbw.description = Shelly Plus RGBW PM - LED Control
 # Plus Devices
 thing-type.shelly.shellyplus1.description = Shelly Plus 1 (Single Relay Switch)
 thing-type.shelly.shellyplus1pm.description =  Shelly Plus 1PM - Single Relay Switch with Power Meter
+thing-type.shelly.shellyplus1l.description = Shelly Plus 1L (Single Relay Switch)
 thing-type.shelly.shellyplus2pm-relay.description = Shelly Plus 2PM - Dual Relay Switch with Power Meter
 thing-type.shelly.shellyplus2pm-roller.description = Shelly Plus 2PM - Roller Control with Power Meter
+thing-type.shelly.shellyplus2l.description = Shelly Plus 2L - Dual Relay Switch
```

Comment:
Ultra minor, any reason the `shellyplus1l` ahsa the single relay switch in braces and the `shellyplus2l` not ?

Reply:
Good catch, there is some inconsistency there. But either way, to make this consistent, we would need to change other texts as well since both forms are used interchangeably.